dyn_em/module_initialize_ideal.F

```diff
@@ -377,10 +380,9 @@ SUBROUTINE init_domain_rk ( grid &
    END DO
 
    CASE(convrad)
-    mminlu2 = ' '
-    mminlu2(1:4) = 'USGS'
-    CALL nl_set_mminlu(1, mminlu2)
-!   CALL nl_set_mminlu(1, 'USGS')
+
+    mminlu_temp(1:10) = 'never_used'
```

Comment:
@kkeene44 
Kelly,
This is where the original value was "USGS". Let's change your mod from "never used" to "USGS" for this line.

Reply:
@davegill 
I modified this, and the one below that you requested.